### PR TITLE
Add support for latest Solidus versions

### PR DESCRIPTION
**Pull request by jtapia**

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.5.1
```
Comment (jacobherrington) [inline .ruby-version]: I'm not sure if this matters, but we may not want it in VC.

Comment (jtapia) [reply .ruby-version]: my bad, I just removed it

**Comment by spaghetticode** [comment]
@jtapia thank you! I think updating the ruby version in `travis.yml` to 2.4 will fix the [failing builds](https://travis-ci.org/solidusio-contrib/solidus_print_invoice/builds/541441955?utm_source=github_status&utm_medium=notification) with Solidus master: 

```
ActionView::Template::Error:
       undefined method `finite?' for 0:Fixnum
```

**Comment by jtapia** [comment]
@spaghetticode thanks, I updated it to `2.4.6`

**Pull request merged**